src/main/java/ua/tc/marketplace/model/dto/user/CreateUserDto.java

```diff
@@ -0,0 +1,34 @@
+package ua.tc.marketplace.model.dto.user;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+import ua.tc.marketplace.model.entity.ContactInfo;
+import ua.tc.marketplace.model.entity.Photo;
+
+public record CreateUserDto(
+    @Schema(example = "taras@shevchenko.ua")
+    @NotBlank
+    String email,
+
+    @Schema(example = "strong_secure_password_with_bigAndSmallLetters_and_digits_and_symbols")
+    @NotBlank
+    String password,
+
+    @Schema(example = "INDIVIDUAL")
+    @NotBlank
+    String userRole,
+
+    @Schema(example = "Taras")
+    @NotBlank
+    String firstName,
+
+    @Schema(example = "Shevchenko")
+    String lastName, // optional
+
```

Comment:
Here should be not `Photo` but `MultipartFile`

Reply:
I thought a little, and I understand the logic. This is `CreateUserDto`, so it is used on registration form, when new user enters their name and password. They usually dont upload photo on registration step. So field `Photo` in this case is redundant, lets delete it from here. And we decide that user registers without photo and loads it later

Reply:
Well, I also have some new thoughts. First, its beter be Long profilePictureId. Second - people can upload picture, but i think we should separate Photo from the rest of the text info. And this also concern AdController. Remember populare services, you can browse for file, and after that the file is getting uploaded (works separate controller method to upload photos), and only after that you hit the button and all you registration info goes to backend. My point is that "create" endpoint should contain dto without files.

Reply:
we decide to remove `Photo` field from CreateUserDto and UpdateUserDto